packages/@react-spectrum/tabs/chromatic/Tabs.chromatic.tsx

```diff
@@ -0,0 +1,102 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Content} from '@react-spectrum/view';
+import {Heading, Text} from '@react-spectrum/text';
+import {Item, Tabs} from '../';
+import {Meta, Story} from '@storybook/react';
+import React from 'react';
+import {SpectrumTabsProps} from '@react-types/tabs';
+
+const meta: Meta<SpectrumTabsProps<object>> = {
+  title: 'Tabs',
+  component: Tabs,
+  parameters: {chromaticProvider: {disableAnimations: true}}
+  /*
+  parameters: {
+    // Might need a delay for font calculations or tab indicator movement/calculations
+    chromatic: { delay: 300 },
+  }*/
+};
+
+export default meta;
+
+
+const Template = <T extends object>(): Story<SpectrumTabsProps<T>> => (args) => (
+  <Tabs {...args} aria-label="Tab example" maxWidth={500}>
+    <Item title="Tab 1" key="val1">
+      <Content margin="size-160">
+        <Heading>Tab Body 1</Heading>
+        <Text>
+          Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
+        </Text>
+      </Content>
+    </Item>
+    <Item title="Tab 2" key="val2">
+      <Content margin="size-160">
+        <Heading>Tab Body 2</Heading>
+        <Text>
+          Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
+        </Text>
+      </Content>
+    </Item>
+    <Item title="Tab 3" key="val3">
+      <Content margin="size-160">
+        <Heading>Tab Body 3</Heading>
+        <Text>
+          Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
+        </Text>
+      </Content>
+    </Item>
+    <Item title="Tab 4" key="val4">
+      <Content margin="size-160">
+        <Heading>Tab Body 4</Heading>
+        <Text>
+          Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
+        </Text>
+      </Content>
+    </Item>
+    <Item title="Tab 5" key="val5">
+      <Content margin="size-160">
+        <Heading>Tab Body 5</Heading>
+        <Text>
+          Dolore ex esse laboris elit magna esse sunt. Pariatur in veniam Lorem est occaecat do magna nisi mollit ipsum sit adipisicing fugiat ex. Pariatur ullamco exercitation ea qui adipisicing.
+        </Text>
+      </Content>
+    </Item>
+  </Tabs>
+);
+
+
```

Comment:
disabled keys + icons + a collapsed tabs story would be nice to have as well

Reply:
Yeah, I didn't want to go too far because this one the API is about to change